tests/test_spawn.py

```diff
@@ -0,0 +1,94 @@
+import os
+import requests
+import pytest
+import time
+
+# Makes heavy use of JupyterHub's API:
+# http://petstore.swagger.io/?url=https://raw.githubusercontent.com/jupyterhub/jupyterhub/master/docs/rest-api.yml
+
+def test_api(request_data):
+    print("asking for the hub's version")
+    r = requests.get(request_data['hub_url'])
+    r.raise_for_status()
+    print(r.json())
+    return r.json()
```

Comment:
why return something from the test function?

Reply:
Good point :P